packages/core/src/metrics/MetricsService.ts

```diff
@@ -19,10 +19,22 @@ export class MetricsService {
 
   private publisher: MetricsPublisher;
 
+  private readonly configuration: ServiceConfiguration;
   private readonly defaultMetrics: Metrics[];
 
-  constructor(private readonly configuration: ServiceConfiguration) {
-    this.publisher = new NetworkMetricsPublisher(configuration.url);
+  constructor(appConfig: AeroGearConfiguration) {
+    const configuration = new ConfigurationHelper(appConfig).getConfig(MetricsService.ID);
+
+    if (!configuration) {
+      console.info("Metrics configuration is missing. Metrics will not work properly.");
```

Comment:
Replace " Metrics will not work properly" => " Metrics will not be published to remote server"